pkg/project/classic/Taskfile.yml

```diff
@@ -12,77 +12,77 @@ vars:
   APM_VERSION: 5.5.1
 
 tasks:
-
   setup:
-    desc: setup AEM environment
+    desc: sets up the environment with AEM and dispatcher
     cmds:
       - task: aem:setup
-      - task: docker:up
+      - task: dispatcher:up
 
   resetup:
-    desc: destroy and setup again AEM environment
+    desc: destroys and sets up the environment again
     cmds:
       - task: destroy
       - task: setup
 
   start:
-    desc: start AEM environment
+    desc: starts the environment with AEM and dispatcher
     aliases: [up]
     cmds:
       - task: aem:up
-      - task: docker:up
+      - task: dispatcher:up
 
   stop:
-    desc: stop AEM environment
+    desc: stops the environment
     aliases: [down]
     cmds:
-      - task: docker:down
+      - task: dispatcher:down
       - task: aem:down
 
   restart:
+    desc: re-starts environment including AEM and dispatcher
     cmds:
       - task: aem:down
       - task: aem:up
+      - task: dispatcher:down
+      - task: dispatcher:up
 
   destroy:
-    desc: destroy AEM environment
+    desc: destroys the entire environment
     cmds:
       - task: stop
       - task: aem:destroy
+      - task: dispatcher:destroy
 
   aem:setup:
-    desc: setup AEM instances
+    desc: sets up AEM instances by starting and provisioning them. It also deploys the code to them
     cmds:
       - task: aem:start
       - task: aem:provision
       - task: aem:deploy
 
-  aem:create:
-    desc: create AEM instances
-    cmds: [sh aemw instance create]
-
   aem:start:
-    desc: start AEM instances
+    desc: starts AEM instances
     aliases: [aem:up]
     cmds:
       - sh aemw instance create
       - sh aemw instance up
 
   aem:await:
-    desc: await stable AEM instances
+    desc: awaits stable AEM instances
+    internal: true
```

Comment:
this is sth useful when called standalone

Reply:
Hmm - I'm not sure if we should expose these granular actions through tasks for other systems to use it. This whole Task tool was rather to combine actions...